docs/data.rst

```diff
@@ -251,6 +251,15 @@ Example
 
       curl -X GET https://api.ona.io/api/v1/data/328.json?page=1&page_size=4
 
+*The response contains a Link header for easier navigation through the pages.*
+
```

Comment:
We could add a reference to the RFC on the link header text. See the GitHub docs references on this for inspiration.